<launch>
  <include file="$(find husky_gazebo)/launch/husky_empty_world.launch">
    <!-- <arg name="world_name" value="worlds/robocup14_spl_field.world" /> -->
    <arg name="world_name" value="$(find husky_highlevel_controller)/worlds/singlePillar.world" />
    <arg name="laser_enabled" value="true" />
  </include>

  <!-- <node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="teleop_twist_keyboard" /> -->

  <node pkg="husky_highlevel_controller" type="husky_highlevel_controller.py" name="husky_highlevel_controller">
    <!-- We can set the parameters individually, or -->
    <!-- <rosparam param="scan_topic">"scan"</rosparam> -->
    <!-- <rosparam param="queue_size">1</rosparam> -->

    <!-- Load from config file -->
    <rosparam command="load" file="$(find husky_highlevel_controller)/config/config.yaml" />
  </node>
</launch>
